ol3-viewer/src/ome/ol3/controls/ScaleBar.js

```diff
@@ -14,21 +14,31 @@ ome.ol3.controls.ScaleBar = function(opt_options) {
     if (typeof(opt_options) !== 'object') opt_options = {};
 
     /**
-	 * list available units for display incl. symbol and
-     * micron multiplicator
-	 *
-	 * @type {Array.<objects>}
-	 * @private
-	 */
+     * list available units for display incl. symbol, threshold for usage
+     * and micron multiplication factor
+     *
+     * @type {Array.<objects>}
+     * @private
+     */
     this.UNITS = [
-        { unit: 'micron', multiplier: 1, symbol: 'µm'},
-        { unit: 'millimeter', multiplier: 1000, symbol: 'mm'}];
+        { unit: 'angstrom',
```

Comment:
This should be moved to the utils package since we will need to use the units in the new info tab (I will open a PR)